src/managed_data/api.py

```diff
@@ -95,7 +95,7 @@ def get_cell_data(
     row_id: str,
     column_name: str,
     client: Optional[Client] = None,
-):
+) -> Any:
```

Comment:
absolutely. i like to tightly type everything. the reason for this is it's actually not clear to me what the type of this is at all -- it can be anything that is contained in a cell, which is still evolving